net-libs/pjproject/pjproject-2.15.1-r1.ebuild

```diff
@@ -97,6 +97,9 @@ src_configure() {
 
 	[ "${videnable}" = "--enable-video" ] && _pj_set_define PJMEDIA_HAS_VIDEO 1 || _pj_set_define PJMEDIA_HAS_VIDEO 0
 
+	# bug 955077 and bug 955129
+	use libyuv && myconf+=( --with-external-yuv )
```

Comment:
Does this mean it was automagic w/ USE=-libyuv when libyuv was installed? (If so, revbump is needed).

Reply:
No, its handled in up in the VIDEO_CARDS use stuff. I just didn't expect (and try to test) if --with-external-yuv overrided the --disable-libyuv that it would generate with USE=-libyuv.

Reply:
ok